src/tests/utils.test.ts

```diff
@@ -0,0 +1,53 @@
+import { describe, it, expect, beforeEach, vi } from 'vitest'
```

Comment:
I don't have a strong opinion here, but in so much as it'd be nice to be consistent with other SI repos, I believe folks are:

* Using `./tests` for Playwright tests
* Organizing unit tests files alongside code files (e.g. this would live as `./src/lib/utils.test.ts`) in lieu of a `./src/tests` directory